src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -67,6 +67,7 @@
                          (rf/dispatch [:wallet/clean-selected-token])
                          (rf/dispatch [:wallet/clean-selected-collectible])
                          (rf/dispatch [:navigate-back]))]
+    (rn/use-unmount #(rf/dispatch [:wallet/clean-selected-token]))
```

Comment:
When going back from the select asset screen we need to clean the selected token

Reply:
yep this is the main thing! 🙏